authors/apps/authentication/tests/test_auser_account.py

```diff
@@ -11,20 +11,6 @@ class TestUser(BaseTestCase):
         Holds user account tests
     """
 
-    def test_new_user_can_register(self):
-        """
-            A user should be informed of a verification email sent to
```

Comment:
why do we need to do away with this test

Reply:
The test is still there I just transferred it to file `test_amail_send` I thought my test was depending on it

Reply:
Great...